pages/common/git-tag.md

```diff
@@ -22,3 +22,7 @@
 - Get updated tags from upstream:
 
 `git fetch --tags`
+
+- List tags that contain a given commit:
```

Comment:
Confirmed: http://stackoverflow.com/questions/36246615/how-does-git-tag-contains-work

As for the ability to add multiple tags to a commit, I found nothing suggesting such a restriction in neither https://git-scm.com/book/en/v2/Git-Basics-Tagging nor https://git-scm.com/docs/git-tag.